src/main/java/com/microsoft/graph/content/MSBatchRequestStep.java

```diff
@@ -57,7 +58,7 @@ public Request getRequest() {
      * @return the list of steps this step depends on
      */
     @Nullable
-    public List<String> getArrayOfDependsOnIds(){
-        return arrayOfDependsOnIds;
+    public HashSet<String> getDependsOnIds(){
+        return dependsOnIds;
```

Comment:
breaking public signature change

Reply:
that's ok, we're on a major bump anyway, now is our chance! but thanks for letting me know

Reply:
sounds good. would be nice to add it into release notes.